loader/wsi.c

```diff
@@ -251,9 +251,11 @@ VKAPI_ATTR VkResult VKAPI_CALL terminator_GetPhysicalDeviceSurfaceSupportKHR(VkP
     *pSupported = false;
 
     if (NULL == icd_term->dispatch.GetPhysicalDeviceSurfaceSupportKHR) {
+        // set pSupported to false as this driver doesn't support WSI functionality
+        *pSupported = false;
         loader_log(loader_inst, VULKAN_LOADER_ERROR_BIT, 0,
```

Comment:
You're going to hate me.... But, shouldn't all these now be VULKAN_LOADER_WARNING_BIT?

Reply:
I used ERROR because the application doesn't have any way to know that the call will fail, and it needs to be a pretty loud 'error'. For the most part this is a rare case, but this is the step below abort/assert but above a 'warning' log message.